kvrocks.conf

```diff
@@ -794,6 +794,14 @@ rocksdb.compression_level 32767
 # Default: 2 MB
 rocksdb.compaction_readahead_size 2097152
 
+# Disable compression for first n levels.
+# By default disabled for the first two levels, because it may contain the
+# frequently accessed data, so it'd be better to use uncompressed data to
+# save the CPU.
+#
+# Default: 2
+rocksdb.nocompression_for_first_levels 2
```

Comment:
After some rethinking, maybe it can be `nocompression_for_first_n_levels`, since `first` usually be just one element.

Reply:
ok, i will fix it

Reply:
fixed